src/shared/populateFilePaths.ts

```diff
@@ -84,10 +94,9 @@ export const populateFilePaths = (elements: ChangeResult[], packageDirPaths: str
         ...elementMap.get(key),
         modified: true,
         origin: 'remote',
-        filenames: [matchingComponent.xml as string, ...matchingComponent.walkContent()].filter((filename) => filename),
+        filenames: getAllFiles(matchingComponent),
```

Comment:
several places use [xml, ...walkContent] and then have filter out non-strings, so that's now shared fn